fbgemm_gpu/codegen/embedding_backward_code_generator.py

```diff
@@ -45,6 +45,7 @@
 # An optimization for ROCm
 env.globals["items_per_warp"] = 128 if args.is_rocm is False else 256
```

Comment:
If we have use_rocm being exposed to jinja, we can probably avoid needing an extra "items_per_warp"